stl/inc/atomic

```diff
@@ -1843,6 +2045,47 @@ _Ty atomic_fetch_xor_explicit(
     return _Mem->fetch_xor(_Value, _Order);
 }
 
+template <class _Ty>
+void atomic_wait(const volatile atomic<_Ty>* _Mem, typename atomic<_Ty>::value_type _Expected) {

```

Comment:
Every parameter of the functions in this chunk of green could be `const`.

Reply:
It would be not exactly as in [p1135](wg21.link/p1135) then.
I understand that for simple types used here it will not affect signature, but what's the reason?

Reply:
Codebase is `const` everywhere even for value parameters.

Example from a quick search:
https://github.com/microsoft/STL/blob/master/stl/inc/algorithm#L433

Reply:
Yeah, passing value parameters by const is our preferred modern convention. Older code doesn't follow it rigorously, and even in newer code we're sometimes lazy in one-line functions, but const everywhere is the ideal and the goal.

This helps prevent mistakes (of accidentally modifying parameters). Also, when consistently followed, it makes non-const parameters (and local variables, etc.) stand out at the time of declaration as "hey, this is something that's going to change, watch out".